src/server/web_server.js

```diff
@@ -37,53 +38,54 @@ const stats_aggregator = require('./system_services/stats_aggregator');
 const addr_utils = require('../util/addr_utils');
 const kube_utils = require('../util/kube_utils');
 const http_utils = require('../util/http_utils');
+const server_rpc = require('./server_rpc');
 
 const rootdir = path.join(__dirname, '..', '..');
 const dev_mode = (process.env.DEV_MODE === 'true');
+const http_port = process.env.PORT || '5001';
+const https_port = process.env.SSL_PORT || '5443';
```

Comment:
maybe worth moving it to the config.js? 
so it will be something like this: 
```suggestion
const http_port = config.PORT;
const https_port =config.SSL_PORT;
```